src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -74,23 +74,39 @@ def load_arguments(self, _):
+                   help="Date or datetime after which credentials expire (e.g. '2017-12-31T11:59:59+00:00' or "
+                        "'2017-12-31'). Default value is one year after current time")
         c.argument('key_value', arg_group='keyCredentials',
                    help='the value for the key credentials associated with the application')
         c.argument('key_type', arg_group='keyCredentials',
-                   help='the type of the key credentials associated with the application', arg_type=get_enum_type(['AsymmetricX509Cert', 'Password', 'Symmetric'], default='AsymmetricX509Cert'))
+                   help='the type of the key credentials associated with the application',
+                   arg_type=get_enum_type(['AsymmetricX509Cert', 'Password', 'Symmetric'],
+                                          default='AsymmetricX509Cert'))
         c.argument('key_usage', arg_group='keyCredentials',
-                   help='the usage of the key credentials associated with the application.', arg_type=get_enum_type(['Sign', 'Verify'], default='Verify'))
-        c.argument('credential_description', help="the description of the password")
-
-        c.argument('required_resource_accesses', type=validate_file_or_dict,
-                   help="resource scopes and roles the application requires access to. Should be in manifest json format. See examples below for details")
-        c.argument('app_roles', type=validate_file_or_dict,
-                   help="declare the roles you want to associate with your application. Should be in manifest json format. See examples below for details")
-        c.argument('optional_claims', type=validate_file_or_dict,
-                   help="declare the optional claims for the application. Should be in manifest json format. See examples below for details. Please reference https://docs.microsoft.com/azure/active-directory/develop/active-directory-optional-claims#optionalclaim-type for optional claim properties.")
+                   help='the usage of the key credentials associated with the application.',
+                   arg_type=get_enum_type(['Sign', 'Verify'], default='Verify'))
+        c.argument('key_display_name', arg_group='keyCredentials',
+                   help="Friendly name for the key.")
+
+        # JSON properties
+        json_property_help = "Should be in manifest JSON format. See examples below for details"
+        c.argument('required_resource_accesses', arg_group='JSON property', type=validate_file_or_dict,
+                   help="Specifies the resources that the application needs to access. This property also specifies "
+                        "the set of delegated permissions and application roles that it needs for each of those "
+                        "resources. This configuration of access to the required resources drives the consent "
```

Comment:
We need to update argument help messages according to the new MS Graph API, such as https://docs.microsoft.com/en-us/graph/api/resources/application

![image](https://user-images.githubusercontent.com/4003950/157584636-eea1334c-0368-4c4c-b5c6-7958f598eb48.png)